process/README.md

```diff
@@ -61,7 +61,7 @@ Incubating projects are required to undergo due diligence as a part of the proce
 Criteria: 
 To be accepted to incubating stage, a project must meet the sandbox stage requirements plus:
 
-* Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+* Document that it is being used successfully in production by at least three independent adopters which, in the TOC’s judgement, are of adequate quality and scope.
```

Comment:
With the wider set of organisations that would qualify as adopters, and with the growth of cloud native in general, is three adopters really enough to be confident in the project?

Reply:
@lizrice we have to balance that concern with the time taken to conduct the interviews as well .. worth noodling over.

Reply:
we also say "at least three" ... (not utmost three!) there have been recent promotions where we have talked to more than 3 for sure (for example i remember doing 4 for kyverno)

Reply:
Also worth bearing in mind that the easier the minimum set of criteria is, the sooner projects will apply for incubation (which was, historically at least, supposed to be a high bar for inclusion)

Reply:
Recommend discussion on altering 3 versus 4 or more adopters be on a separate issue as it has wider impact than the intended clarification updates in this PR.